User/data-access/UserDAO.ts

```diff
@@ -0,0 +1,38 @@
+import { UserModel } from '../models/UserModel';
+import { User } from '../types/userTypes';
+
+export class UserDAO {
+
+	public static async getAllUsers(): Promise<UserModel[]> {
+		return UserModel.findAll({
+			where: {'deletedAt': null},
```

Comment:
Actually, you even don't need to specify `deletedAt` here and in other places, because `sequelize` takes care of that and will not return deleted records (records with filled deletetAt).